<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">selection-background-color: rgb(0, 170, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 49, 97);
font: 75 16pt &quot;Verdana&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Citas</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>771</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabWidgetPage1">
         <attribute name="title">
          <string>Añadir</string>
         </attribute>
         <widget class="QPushButton" name="btnAgregaU">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>480</y>
            <width>281</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Registrar</string>
          </property>
         </widget>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>871</width>
            <height>241</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QCalendarWidget" name="calendarWidget"/>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="0">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Fecha y hora:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Motivo:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDateTimeEdit" name="dateTimeEdit"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_11"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>ID Mascota</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lineEdit_13"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>ID veterinario</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEdit_15"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>761</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 36pt &quot;Comic Sans&quot;;</string>
          </property>
          <property name="text">
           <string>Añadir Cita</string>
          </property>
         </widget>
         <widget class="QTableWidget" name="tableWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>360</y>
            <width>961</width>
            <height>101</height>
           </rect>
          </property>
          <row>
           <property name="text">
            <string>disponible</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>9:00</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>10:00</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>11:00</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>12:00</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>13:00</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>14:00</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>16:00</string>
           </property>
          </column>
         </widget>
         <widget class="QPushButton" name="btnRegresar">
          <property name="geometry">
           <rect>
            <x>536</x>
            <y>480</y>
            <width>311</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Volver al Menú</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tabWidgetPage2">
         <attribute name="title">
          <string>Eliminar</string>
         </attribute>
         <widget class="QSplitter" name="splitter">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>340</y>
            <width>125</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Ingrese ID a eliminar:</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="btnAgregaU_2">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>340</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>20</y>
            <width>332</width>
            <height>72</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 36pt &quot;Comic Sans&quot;;</string>
          </property>
          <property name="text">
           <string>Eliminar Cita</string>
          </property>
         </widget>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>110</y>
            <width>631</width>
            <height>201</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>ID Cita</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID Vet</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Motivo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Fecha</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Hora</string>
           </property>
          </column>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>340</y>
            <width>137</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btnRegresar_2">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>390</y>
            <width>311</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Volver al Menú</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tabWidgetPage3">
         <attribute name="title">
          <string>Actualizar</string>
         </attribute>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>90</y>
            <width>191</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Ingrese ID del la cita a actualizar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>10</y>
            <width>471</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 36pt &quot;Comic Sans&quot;;</string>
          </property>
          <property name="text">
           <string>Actualizar Cita</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>120</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btnAgregaU_3">
          <property name="geometry">
           <rect>
            <x>520</x>
            <y>120</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>120</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>440</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Actualizar</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnRegresar_4">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>440</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Volver al Menú</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget_4">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>170</y>
            <width>470</width>
            <height>236</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_12"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>ID Mascota</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit_14"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>ID veterinario</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Motivo:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Fecha y hora:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_16"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tabWidgetPage4">
         <attribute name="title">
          <string>Conultar</string>
         </attribute>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>0</y>
            <width>128</width>
            <height>72</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 36pt &quot;Comic Sans&quot;;</string>
          </property>
          <property name="text">
           <string>Citas</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>80</y>
            <width>137</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>80</y>
            <width>64</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Ingrese ID:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>590</x>
            <y>80</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
         <widget class="QTableWidget" name="tableWidget_4">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>160</y>
            <width>631</width>
            <height>201</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>ID Cita</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID Vet</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Motivo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Fecha</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Hora</string>
           </property>
          </column>
         </widget>
         <widget class="QPushButton" name="btnRegresar_3">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>420</y>
            <width>311</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Volver al Menú</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
